photon-ml/src/main/scala/com/linkedin/photon/ml/util/PalDBIndexMap.scala

```diff
@@ -157,7 +158,8 @@ class PalDBIndexMap extends IndexMap {
 
   //noinspection ScalaStyle
   override def -(key: String): Map[String, Int] = {
-    throw new RuntimeException("This map is a read-only immutable map, remove operation is unsupported.")
+    throw new RuntimeException("This map is a read-only immutable map, " +
+      "remove operation is unsupported.")
```

Comment:
It appears to my IDE that it does not fit in one line (length of this line == 106).